cmd/commands/server.go

```diff
@@ -15,14 +15,14 @@ import (
 // ServerCmd ...
 func ServerCmd() *cobra.Command {
 	var port uint = 22
-	var address string = "0.0.0.0"
+	var address = "0.0.0.0"
```

Comment:
suggestion: use shorthand for all variables

example

```go
idleTimeout := uint(0)
hostKeyFile := cssh.DefaultHostKeyFile
```

Reply:
what do you think about `var port uint = 22` vs `port := uint(22)`?

Reply:
Have left those as-is for now, so we can merge this.  "cleanup" is never-ending process :)

Reply:
I think type inferred declarations are preferred in general. There's a lot of places in the code that can be updated to use the shorthand for sure

Reply:
Not as many as I thought. Only found 9 further matches for `r/var \w+ \w+ =/` mostly of the uint pattern above. Updating now to uint(x) version.

![Screen Shot 2021-10-03 at 4 42 50 pm](https://user-images.githubusercontent.com/122371/135741621-b25bebc9-f83f-428c-a997-37806d41c967.png)

Reply:
Pushed a change for this - hope I understand correctly!